### ISSUE TITLE
Configure VR sessions for user study and manage those configurations

### ISSUE THREAD

**Issue by dennisgbrown** [AugCogVR/cogbre-vr#37]
For the user study, the VR system will:
1) Only show the relevant collection and binary file(s) to the user 
2) Disable certain features based on the testing condition
3) Need to know the alias/identifier of the testing participant before the session starts

Thus it will need to be reconfigured for each user study session.

Simplest but most inelegant solution is to make the VR system read a config file before starting the session, and we manually edit or swap out the config file before starting the session.

Most elegant but expensive solution is to add these configuration options within the VR itself. Alternative idea, also expensive, is to build a simple 2D "control panel" app that communicates config info to the VR app. 

I am fine starting with the config file method and we can grow from there when justified.

**Comment by dennisgbrown** [comment]
For the user study, the VR system will:
1) Only show the relevant collection and binary file(s) to the user 
2) Disable certain features based on the testing condition
3) Need to know the alias/identifier of the testing participant before the session starts

Simplest but most inelegant solution is to make the VR system read a config file before starting the session, and we manually edit or swap out the config file before starting the session.

Most elegant but expensive solution is to add these configuration options within the VR itself. Alternative idea, also expensive, is to build a simple 2D "control panel" app that communicates config info to the VR app. 

I am fine starting with the config file method and we can grow from there when justified.

**Comment by dennisgbrown** [comment]
Another concern -- maybe it needs to be its own issue -- is, once we know what configuration the VR is supposed to run, HOW do we enable/disable functionality in the VR. For example, I want to disable the option to arbitrarily move slates in space. But instead, we need to give the user something that could still work, such as spawning new slates into preset positions in space (e.g., a spherical grid surrounding the user).
